docs-source/content/faq/namespace-management.md

```diff
@@ -67,7 +71,7 @@ $ helm upgrade \
   kubernetes/charts/weblogic-operator
 ```
 
-{{% notice note %}}
+d{{% notice note %}}
```

Comment:
Additional "d" added, which breaks the notice.

I recommend double checking all of your changes using hugo (scripts at top of docs-source).

Reply:
bad keystroke, fixed